client/my-sites/domains/domain-management/edit/domain-types/registered-domain-type.jsx

```diff
@@ -345,6 +345,28 @@ class RegisteredDomainType extends React.Component {
 			'domains/new-status-design/new-options'
 		);
 
+		let expiresText;
```

Comment:
Might be good to just move this whole block to a separate function - then you'll keep `render` function smaller and you'll be able to assign the return value from the new function to a `const` variable 👍

Reply:
Good idea!